test/phoenix/live_dashboard/pages/applications_page_test.exs

```diff
@@ -39,25 +39,30 @@ defmodule Phoenix.LiveDashboard.ApplicationsPageTest do
   end
 
   test "not started applications have different status in table" do
+    Application.stop(:ssh)
+    Application.unload(:ssh)
+
     {:ok, live, _} = live(build_conn(), applications_path(50, "", :version, :asc))
     rendered = render(live)
 
     assert rendered =~ ~s|<tr id="app-stdlib">|
     refute rendered =~ ~s|<tr id="app-ssh"|
 
-    Application.load(:ssh)
+    assert :ok = Application.load(:ssh)
+
     {:ok, live, _} = live(build_conn(), applications_path(50, "", :version, :asc))
     rendered = render(live)
     assert rendered =~ ~s|<tr class="text-muted" id="app-ssh"|
 
-    Application.start(:ssh)
+    assert :ok = Application.start(:ssh)
     {:ok, live, _} = live(build_conn(), applications_path(50, "", :version, :asc))
     rendered = render(live)
 
     assert rendered =~
              ~s|tr phx-click="show_info" phx-value-info="App&lt;ssh&gt;" phx-page-loading="phx-page-loading" id="app-ssh">|
 
-    Application.unload(:ssh)
+    assert :ok = Application.stop(:ssh)
+    assert :ok = Application.unload(:ssh)
```

Comment:
I don't know why, but the ssh application is now started by default in the CI, so I manually stopped before running the test. Feel free to solve this problem differently.

Reply:
Ok, I think this PR introduced the ssh application: https://github.com/hexpm/hex/pull/974
And this one fixed: https://github.com/hexpm/hex/pull/977

But we have to wait for a new hex release 😅 
Then I can revert this commit, I guess